### ISSUE TITLE
TestBasicExtractor.testExtractColumnsCorrectly3: Fix different heights calculation

### ISSUE THREAD

**Issue by melisabok** [melisabok/tabula-java#3]
Different instances of PDFont.

TestBasicExtractor.testExtractColumnsCorrectly3

old: org.apache.pdfbox.pdmodel.font.PDType1Font
new: org.apache.pdfbox.pdmodel.font.PDType1CFont

**Comment by melisabok** [comment]
Fixed expected table definition, it should be 5 columns instead of 6. Added more tests for the same pdf file

https://github.com/melisabok/tabula-java/commit/fb67fb6a1945a551e4d76dc0a0be103607ae9bd9

**Comment by jazzido** [comment]
New expectation is definitely correct, but can't figure out which change corrected that. Anything to do with the fonts?

**Comment by melisabok** [comment]
Yes, the calculated font height for all the text elements are different. 

<img width="1208" alt="screen shot 2017-03-07 at 8 49 19 am" src="https://cloud.githubusercontent.com/assets/545506/23659124/1fa8b074-0313-11e7-8041-3681138c412d.png">

**Comment by melisabok** [comment]
Here are the outputs

[textElements_new.txt](https://github.com/melisabok/tabula-java/files/824570/textElements_new.txt)
[textElements_old.txt](https://github.com/melisabok/tabula-java/files/824571/textElements_old.txt)
